src/dsymbol.d

```diff
@@ -835,6 +819,13 @@ public:
         return p ? p.isAggregateDeclaration() : null;
     }
 
+    // is this a member of a ClassDeclaration?
```

Comment:
`Dsymbol.isClassMember` is now a specialized version of `Dsymbol.isMember`. Moving the function is good to represent those relations.